cytotable/presets.py

```diff
@@ -317,6 +317,33 @@
         # and modified at runtime as needed
         "CONFIG_JOINS": "",
     },
+    "deepprofiler": {
+        # version specifications using related references
+        "CONFIG_SOURCE_VERSION": {
+            "deepprofiler": "v0.3.1",
+            "cellprofiler": "v4.2.x",
+        },
+        # names of source table compartments (for ex. cells.csv, etc.)
+        "CONFIG_NAMES_COMPARTMENTS": tuple(),
```

Comment:
you might consider adding why these are not needed?

Reply:
Thanks - will do!